base_layer/core/src/validation/block_body/block_body_internal_validator.rs

```diff
@@ -80,23 +80,18 @@ fn validate_block_specific_checks(
     consensus_manager: &ConsensusManager,
     factories: &CryptoFactories,
 ) -> Result<(), ValidationError> {
-    let constants = consensus_manager.consensus_constants(block.header.height);
     if block.header.height == 0 {
         warn!(target: LOG_TARGET, "Attempt to validate genesis block");
         return Err(ValidationError::ValidatingGenesis);
     }
     check_coinbase_output(block, consensus_manager, factories)?;
-    check_coinbase_output_features(&block.body, constants)?;
+    check_coinbase_output_features(&block.body)?;
 
     Ok(())
 }
 
-fn check_coinbase_output_features(
-    body: &AggregateBody,
-    consensus_constants: &ConsensusConstants,
-) -> Result<(), ValidationError> {
-    let max_coinbase_metadata_size = consensus_constants.coinbase_output_features_extra_max_length();
-    body.check_output_features(max_coinbase_metadata_size)
```

Comment:
we should still check this

Reply:
You are breaking the consensus pattern here. 
We must have a MAX limit of bytes allowed by the vector. 
BUT that should be independent of the consensus value. The consensus value can allow a lower limit